about.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <script src="https://kit.fontawesome.com/cb6c5078b1.js"
+      crossorigin="anonymous"
+    ></script>
+     <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"/>
+  <link rel="stylesheet" href="about.css">
+  <title>About Securam Entertainment</title>
+</head>
+<body>
+  
+<header>
+    <aside class="top-section">
+      <div class="bar-container">
+      <ul class="bars">
+        <li>
+          <i class="fab fa-facebook-f"></i>
+        </li>
+        <li>
+          <i class="fab fa-twitter"></i>
+        </li>
+        <li>
+          <h4>English</h4>
+        </li>
+        <li>
+          <h4>My Page</h4>
+        </li>
+        <li>
+          <h4>Logout</h4>
+        </li>
+      </ul>
+      </div>
+    </aside>
+    <div class="navbar">
+        <div class="logo-DT">
+          <a href="index.html"><img src="./image/logo.jpg" alt="LOGO" class="logo"></a>
+          <div class="menu-bar nav" id="menu">
+            <div class="icon"></div>
+            <div class="icon"></div>
+            <div class="icon"></div>
+          </div>
+          <div class="cancel" id ='close'>
+            <p>&times;</p>
+          </div>
+        </div>
+
+        <nav class="menu-list" id="nav-mobile">
+          <ul class="items">
+            <li class="home">
+              <a href="index.html">Home</a>
+            </li>
+
+            <li class="links">
+              <a href="about.html">About</a>
+            </li>
+            <li class="links">
+              <a href="#">Program</a>
+            </li>
+            <li class="links">
+              <a href="#">Join</a>
+            </li>
+            <li class="links">
+              <a href="#">Sponsor</a>
+            </li>
+            <li class="links">
+              <a href="#">News</a>
+            </li>
+          </ul>
+          <div class="company">
+            <a href="index.html">Securam Circle</a>
+          </div>
+        </nav>
+    </div>
+  </header>
+  <section class="first-section">
+    <div class="first-section-container">
+      <h1 class="open-heading">Securam Entertainment</h1>
+      <div class="first-section-para">
+        <p>
+          The leading entertainment platform in Nigeria, unifying the music industry in other to stand firm in global entertainment. This is the very first show bringing the leading artists in Africa together shearing their challenges and how we can continue to put Nigeria in the global space.
+        </p>
+      </div>
+      <div class="event-details">
+        <p>Please contact us per email for any further questions.
```

Comment:
- [ ] Please change the color of this part in order to match the original:

Yours:
![Screenshot from 2021-11-18 19-29-18](https://user-images.githubusercontent.com/62705947/142475437-90f79c7a-fe6a-42e2-9e7f-e9a755005b5a.png)



Original

![Screenshot from 2021-11-18 19-29-14](https://user-images.githubusercontent.com/62705947/142475453-4f0daf93-6fc6-4594-874c-61c9a4a093b3.png)